scene.py

```diff
@@ -0,0 +1,138 @@
+from item import Item
```

Comment:
The way you organized your scenes is very well done - makes it easy to read through and understand!